py-polars/polars/dataframe/frame.py

```diff
@@ -7619,6 +7619,33 @@ def n_unique(self, subset: str | Expr | Sequence[str | Expr] | None = None) -> i
         df = self.lazy().select(expr.n_unique()).collect()
         return 0 if df.is_empty() else df.row(0)[0]
 
+    def approx_unique(self) -> Self:
```

Comment:
We shouldn't expose this on the `DataFrame`. I don't expect this to work with all dtypes, and we don't do this with most expressions.

Reply:
done